pip/commands/list.py

```diff
@@ -91,6 +91,13 @@ def __init__(self, *args, **kw):
             help="Align package names and versions into vertical columns.",
         )
 
+        cmd_opts.add_option(
+            '--leaves',
+            action='store_true',
+            dest='leaves',
+            help="List packages that are not dependencies of installed packages.",
```

Comment:
> OK, cool. BTW, "leaves" doesn't really feel like the right name for "packages that are not dependencies of installed packages". The term "leaves", to me, means "packages with no dependencies", which is what you implemented. 

I have Bash script that does exactly what you mentioned (and that's the reason I created this PR) and for some reason I didn't implement it in the first commit. I guess I should do less things in parallel 😄 
I have fixed the code and changed the name of the option. Let me know what you think.
BTW, thanks for the quick responses!